funnel/forms/organization.py

```diff
@@ -69,7 +69,10 @@ def validate_name(self, field) -> None:
             # from existing name, or has only changed case. This is a validation pass.
             return
         if reason == 'user':
-            if self.user.username and field.data.lower() == self.user.username.lower():
+            if (
+                current_auth.user.username
+                and field.data.lower() == current_auth.user.username.lower()
```

Comment:
Don't access current_auth in a form. The form is supposed to be a data validator that can be used anywhere. Instead, look for uses of the form in views and ensure the form is being given a user.